### ISSUE TITLE
Neural Networks

### ISSUE THREAD

**Issue by revantharunachalam** [python-and-ladder/course-materials#30]
- [ ] Architectures: Perceptrons, CNNs, RNNs, LSTM
- [ ] Activation functions, loss functions, optimizers
